api/APIv2.md

```diff
@@ -245,14 +251,30 @@ Return codes:
 Request:
 
 The request mime type MUST be "application/x-proto-binary".
-The body MUST be a protobuf message of type AuthContainer where the AuthBody is a protobuf message of type [attest.ZAttestReq](./proto/attest/attest.proto).
-`ZAttestReq` has `reqType` which MUST be filled by EVE to identify type of the request. If `reqType` is `ATTEST_REQ_CERT`, `certs` should be filled with certificates used by EVE. If `isIMutable` attribute is not set, Controller should reject any request that would overwrite an existing certificate previously published. If `reqType` is `ATTEST_REQ_NONCE`, no other field needs to be set in `ZAttestReq`. If `reqType` is `ATTEST_REQ_QUOTE`, then `quote` MUST be populated with the quote generated by Trusted Platform Module.
+
+The body MUST be a protobuf message of type AuthContainer where the AuthBody is a protobuf message of type [attest.ZAttestReq](./proto/attest/attest.proto). `ZAttestReq` has `reqType` which MUST be filled by device to identify type of the request.
+
+If `reqType` is `ATTEST_REQ_CERT`, `certs` should be filled with certificates used by device. If `isIMutable` attribute is not set, Controller should reject any request that would overwrite an existing certificate previously published.
+
+If `reqType` is `ATTEST_REQ_NONCE`, no other field needs to be set in `ZAttestReq`.
+
+If `reqType` is `ATTEST_REQ_QUOTE`, then `quote` MUST be populated with the following information: `attestData` containing [TPMS_ATTEST structure defined in TCG](https://trustedcomputinggroup.org/wp-content/uploads/TPM-Rev-2.0-Part-2-Structures-01.38.pdf), `signature` containing signature of `TPMS_ATTEST` by the attestation signing key, `pcr_values` containing values of PCR 0-15, `event_log` containing TPM Event Log (aka measurement logs) with each event entry of the format `TpmEventLogEntry`,  `versions` containing current versions of various software components running on the device(e.g. EVE, UEFI etc), `gps_info` containing GPS co-ordinates of device's geo-location.
+
+If `reqType` is `ATTEST_REQ_TYPE_INFO`, then `info` field must be filled with the following information: `integrity_token` containing integrity_token value given by Controller for the current attestation cycle and `keys` containing keys of type `AttestVolumeKey`, which are the decryption keys used by the device for encryption of its volume(s)
```

Comment:
Nit but couldn't we call this something more specific than "info", since it is really about storing encypted keys?
What is the field called where the device gets back the encrypted key from the controller?

Reply:
It is called AttestVolumeKey in the response. 
So ATTEST_REQ_TYPE_STORE_KEYS would make sense.
Also comment should specify that the assumption is that the conveyed keys are encrypted in a device-specific manner (using a TPM if the device has one).